permissions/models.py

```diff
@@ -87,6 +87,10 @@
         "can_view_users",
         _("Can view users in the whole system"),
     ),
+    (
+        "can_comment_reservations",
+        _("Can comment reservations"),
```

Comment:
perhapsies we could add the "whole system" line to here as well to be consistent and making it clear.

Reply:
Added 👍